sql/index/pilosa/driver.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 
 	pilosa "github.com/pilosa/go-pilosa"
+	"github.com/sirupsen/logrus"
```

Comment:
go-log?

Reply:
not yet inside go-mysql-server. Last news i had was to wait until a logging guide came out